packages/optimizer/README.md

```diff
@@ -62,22 +62,22 @@ There's also a [command line version](../cli/README.md) available:
 $ npx @ampproject/toolbox-cli myFile.html
 ```
 
-## Best Practices
+## Why doesn't my AMP page render faster?
 
-### Regenerate pages at least once a week
+The biggest performance gain results from [removing the AMP boilerplate code](https://amp.dev/documentation/guides-and-tutorials/optimize-and-measure/server-side-rendering/#why-is-it-faster?). However, under some circumstances it's not possioble to remove the boilerplate code:
 
-AMP Optimizer inlines CSS styles required by AMP. To make sure, that the inlined CSS stays in sync with the latest AMP release, we recommend to re-generate pages at least once a week. The good news is, out-of-sync CSS will not break your page, as AMP will check the version of the inlined CSS at runtime and will automatically update it to the latest version.
+* if the`amp-experiment`, `amp-story` or `amp-dynamic-css-classes` components are used. 
```

Comment:
Maybe useful to link to the actual source here? https://github.com/ampproject/amphtml/blob/62a9eab084ccd800d80a371e2cb29cd4f9e8576a/src/render-delaying-services.js#L39-L43

Reply:
done